examples/wasm_node/package.json

```diff
@@ -19,6 +19,7 @@
     "filecoin"
   ],
   "dependencies": {
+    "@chainsafe/bls": "^1.0.0",
```

Comment:
can we put this in dev dependencies? I know these are the tests.. but it makes clear that we are not actually using this.

Reply:
Yep not using this I am removing it.